mountinfo/mountinfo_bsd.go

```diff
@@ -1,43 +1,51 @@
-//go:build (freebsd && cgo) || (openbsd && cgo) || (darwin && cgo)
-// +build freebsd,cgo openbsd,cgo darwin,cgo
+//go:build freebsd || openbsd || darwin
+// +build freebsd openbsd darwin
 
 package mountinfo
 
-/*
-#include <sys/param.h>
-#include <sys/ucred.h>
-#include <sys/mount.h>
-*/
-import "C"
-
 import (
-	"fmt"
 	"reflect"
-	"unsafe"
+	"syscall"
 )
 
+func getInfo(r reflect.Value, a string, b string) string {
+	if r.FieldByName(a) != (reflect.Value{}) {
+		r = r.FieldByName(a)
+	} else {
+		r = r.FieldByName(b)
+	}
+	var bs []byte
+	for i := 0; i < r.Len(); i++ {
+		i8 := r.Index(i).Int()
+		if i8 == 0 {
+			break
+		}
+		bs = append(bs, byte(i8))
+	}
+	return string(bs)
+}
```

Comment:
Ratherthan depending on reflects, can we split `mountinfo_bsd.go` into `mountinfo_{freebsd,openbsd}.go`?

Reply:
Okay, I will create a new PR without reflect.

Reply:
Done by https://github.com/moby/sys/pull/114